analysis/analysis-engine.go

```diff
@@ -6,7 +6,10 @@
 package analysisengine
 
 import (
-	"github.com/letsencrypt/boulder/log"
+	blog "github.com/letsencrypt/boulder/log"
```

Comment:
I'm following the convention Richard set in the refactor PR #48. I don't have the energy to change it.